.buildkite/pipeline.yml

```diff
@@ -0,0 +1,110 @@
+env:
+  MAC_BK_AGENT: eng-prod-us-west-2-mac-arm-macos-build-medium
+
+steps:
+  - name: ':hammer: Build and Unit Test iOS Demo'
+    key: 'unit-test'
+    commands:
+      - . ~/.zshrc
+      - rbenv local 2.7.6
+      - bundle install --path vendor/bundle
+      - xcrun simctl boot "iPhone 14" || true
+      - pod install --verbose --repo-update
+      - bundle exec fastlane test
+      - xcrun simctl shutdown "iPhone 14" || true
+    agents:      
+      queue: ${MAC_BK_AGENT}
+    timeout_in_minutes: 25
+
+  - block: ':whale: Publish to TestFlight'
+    key: 'publish-block'
+    prompt: 'Confirm publishing to TestFlight?'
+    depends_on: 'unit-test'
+
+  - name: ':rocket: Publish demo app to TestFlight'
+    commands:
+      - . ~/.zshrc
+      - rbenv local 2.7.6
+      - bundle install --path vendor/bundle
+      - xcrun simctl boot "iPhone 14" || true
```

Comment:
Do we need this?

Reply:
Removed the sim steps - I think the install steps are needed